charts/events/templates/event-sources.yaml

```
@@ -0,0 +1,18 @@
+{{- if index .Values "argo-events" "enabled" }}
```

Best just to traverse the object
```suggestion
{{- if .Values.argo-events.enabled }}
```

---

yeah, this would be cleaner but helm is not happy with `bad character U+002D '-' `
See also here: https://stackoverflow.com/questions/63853679/helm-templating-doesnt-let-me-use-dash-in-names